packages/driver/cypress/support/defaults.js

```diff
@@ -31,3 +31,9 @@ beforeEach(() => {
     $(cy.state('window')).off()
   } catch (error) {} // eslint-disable-line no-empty
 })
+
```

Comment:
Just curious, if someone used `Cypress.require()` at the root supportFile level, what is the error messaging?

Reply:
Any usage of it outside the `cy.origin()` callback results in [this error](https://github.com/cypress-io/cypress/blob/90648a01f517b743f79e9bd3cc2afbc7859b3283/packages/driver/src/cypress/error_messages.ts#L1458-L1461).

```
`Cypress.require()` can only be used inside the `cy.origin()` callback.
```